config/routes.rb

```diff
@@ -9,9 +9,10 @@
 
   namespace :v0, defaults: { format: 'json' } do
     resources :in_progress_forms, only: [:index, :show, :update, :destroy]
-    resources :letters, only: [:index, :show] do
+    resources :letters, only: [:index] do
       collection do
         get 'beneficiary', to: 'letters#beneficiary'
+        post ':id', to: 'letters#download'
```

Comment:
Given that there is a submit the form, to generate the letter, I think POST makes sense here.  It is the body of a form being submitted for the benefit summary letter.